pages/common/vboxmanage-controlvm.md

```diff
@@ -0,0 +1,36 @@
+# vboxmanage-controlvm
+
+> Change the state and the settings of a currently running virtual machine.
+> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-import>.
+
+- Temporary stops the execution of a virtual machine:
+
+`VBoxManage controlvm {{uuid|vmname}}  pause`
+
+- Resume the execution of a paused virtual machine:
+
+`VBoxManage controlvm {{uuid|vmname}} resume`
+
+- Perform a cold reset on the virtual machine:
+
+`VBoxManage controlvm {{uuid|vmname}} reset`
```

Comment:
```suggestion
`VBoxManage controlvm {{uuid|vm_name}} reset`
```

Reply:
Thanks for the suggestion, I am resolving this, as I have already addressed it in my multi-line suggestion.